README.md

```diff
@@ -35,40 +36,59 @@ which maintains the project and offers professional services around it.
 - Small memory footprint and fast execution
 - Domain-driven design architecture, high-quality code and test coverage
 
-## Install
-Before proceeding, install the following prerequisites:
+## Prerequisites
+
+The following are needed to run Mainflux:
+
+- [Docker](https://docs.docker.com/install/) (version 18.09)
+- [Docker compose](https://docs.docker.com/compose/install/) (version 1.24.1)
 
-- [Docker](https://docs.docker.com/install/)
-- [Docker compose](https://docs.docker.com/compose/install/)
+Developing Mainflux will also require:
+
+- [Go](https://golang.org/doc/install) (version 1.13.3)
+- [Protobuf](https://github.com/protocolbuffers/protobuf#protocol-compiler-installation) (version 3.6.1)
+
+## Install
 
-Once everything is installed, execute the following commands from project root:
+Once the prerequisites are installed, execute the following commands from the project's root:
 
 ```bash
-docker-compose -f docker/docker-compose.yml up -d
+docker-compose -f docker/docker-compose.yml -f docker/aedes.yml up
 ```
 
-This will bring up all Mainflux dockers and inter-connect them in the composition.
+This will bring up the Mainflux docker services and interconnect them.  This command can also be executed using the project's included Makefile:
```

Comment:
It seems that you've added double space after the full stop here: `...them.  This command...`. Please check for the other occurrences, as well.

Reply:
Double spaces removed